### ISSUE TITLE
Key logger works only when site is accessed by PC does not work on mobile chrome browser.

### ISSUE THREAD

**Issue by 0verd0se** [DarkSecDevelopers/HiddenEye#133]
Key logger does not work when site is accessed from mobile browsers

**Comment by usama7628674** [comment]
@0verd0se Which site are you talking about?

**Comment by usama7628674** [comment]
@0verd0se Did you include the following lines in code?
```<script src="keylogger.js"></script>```
```<script src="keylogger.php"></script>```

**Comment by 0verd0se** [comment]
@usama7628674  Instagram one it works perfect when accessed on PC but when I open the link from my mobile browser no keystrokes are recorded expect the id pass . 

and I guess keylogger is not a good idea as ngrok limits connections on free hosting and if I spam my keyboard and then type my is and password I don't get redirected I get an error page on ngrok saying I exceeded my connections

**Comment by usama7628674** [comment]
yes it's limit of ngrok for free version but you can use serveo.

**Comment by usama7628674** [comment]
@0verd0se Make sure your internet connection stable cuz  it's working fine on my side.

**Comment by 0verd0se** [comment]
Let me try again i just tried it once though

**Comment by 0verd0se** [comment]
@usama7628674 Does serveo creates a file like ngrok in Server folder? I tried multiple times but the link is not being created and there is no file for serveo like ngrok

**Comment by usama7628674** [comment]
It'll create link.url in main directory.

**Comment by 0verd0se** [comment]
Yeah i got it the link.url ts blank even after waiting 5 mins no error shown

**Comment by usama7628674** [comment]
Did you clone master branch?

**Comment by usama7628674** [comment]
I'll look into it later.Currently busy studying.

**Comment by An0nUD4Y** [comment]
@0verd0se  The keylogger works using javascript codes , As I have tested in mobile chrome, That keylogger file of java script was not loading. So currently its not recording the keystrokes  for Android chrome, Because chrome Don't allow to background javascript as it thought, So wait sometime ,whenever i got time, I'll search for its solution. And if you want you use the tool then  you can use it without keylogger , KEYLOGGER IS CURRENTLY IN TESTING STAGE...

**Comment by 0verd0se** [comment]
@An0nUD4Y No issues, i just wanted to inform that its not working :D

**Comment by An0nUD4Y** [comment]
@0verd0se  Thanks to inform !! You can Help, By making some contribution.?
